Exiled.API/Features/Pickups/Scp244Pickup.cs

```diff
@@ -82,7 +82,7 @@ public Scp244State State
         }
 
         /// <summary>
-        /// Gets or sets the current size percent of the Scp244's Hypothermia.
+        /// Gets or sets the ActivationDot.
```

Comment:
Okay, it's `Gets or sets the activation angle, where 1 is a minimum, and -1 it's a maximum activation angle` something like this, idk better way to say in docs about vectors dot product